examples/computer_vision/unets_tf_keras/model_def.py

```diff
@@ -60,9 +61,10 @@ def download_weights(self):
         weights_dir = self.download_directory + '/weights/'
         data_file = self.context.get_data_config()['data_file']
         mobilenet_link = 'https://storage.googleapis.com/tensorflow/keras-applications/mobilenet_v2/' + data_file
-        os.mkdir(weights_dir)
+        os.makedirs(weights_dir, exist_ok=True)
 
-        urllib.request.urlretrieve(mobilenet_link,weights_dir + data_file)
+        with filelock.FileLock(os.path.join(weights_dir, "download.lock")):
```

Comment:
I think in the example I linked you, there was a brief comment above the filelock usage:

```python
    # Use a file lock so only one worker on each node does the download.
```

Can you put that here too?  I think one of the AMLEs requested that we keep an explanation in a comment of what the filelock was doing.  The assumption is that, while filelocks may be the right tool for the job, they're not common tools in the everyday ML developer's toolbox.